app/controllers/enrollments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class EnrollmentsController < ApplicationController
+  def index
+    @events = Event.where(id: Enrollment.where(created: true).where.not(user_id: @current_user[:id]).pluck(:event_id))
+    @enrolled_events = Enrollment.where(user_id: @current_user[:id], created: false).pluck(:event_id)
```

Comment:
Can we use @current_user.entrollments.pluck(:event_id)  ?

Reply:
Yes, we can